ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -434,40 +468,101 @@ Below you will find two examples of documentation automatically created by Ansib
+
+### Using Ansible AVD with direct eAPI connectivity to the switches
+
+If you want to push to switches in the entire FABRIC using eAPI and your playbook looks as follows:
+
+```yml
+---
+- name: Run AVD
+  hosts: FABRIC
+  gather_facts: false
+  tasks:
+    - name: generate intended variables
+      import_role:
+        name: arista.avd.eos_designs
+    - name: generate device intended config and documentation
+      import_role:
+        name: arista.avd.eos_cli_config_gen
+    - name: deploy configuration to device
+      import_role:
+         name: arista.avd.eos_config_deploy_eapi
+```
+
+You would execute it using the following command:
 
 ```shell
-ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml --tags build
+ansible-playbook playbook.yml
 ```
 
-This will execute the playbook `playbooks/dc1-fabric-deploy-cvp.yml` with the tag `build`. This instructs Ansible to only run the tasks tagged 'build'
-in the playbook - which in this case will result in configuration files being generated but not deployed to devices.
+This will:
+
+- Create a text file containing the device's entire configuration.
+- Create device-specific and fabric-wide documentation.
+- Push the relevant configuration to each device using eAPI.
+
+### Using Ansible AVD in accordance with CloudVision
 
-To push the configuration to devices using CloudVision:
+If you want to push to switches in the entire FABRIC using CloudVision and your playbook looks as follows:
+
+```yml
+---
+- name: Run AVD
+  hosts: FABRIC
+  gather_facts: false
+  tasks:
+    - name: generate intended variables
+      import_role:
+        name: arista.avd.eos_designs
+    - name: generate device intended config and documentation
+      import_role:
+        name: arista.avd.eos_cli_config_gen
+    - name: deploy configuration to device
+      import_role:
+         name: arista.avd.eos_config_deploy_eapi
+
+- name: Push to CVP
+  hosts: CVP
+  gather_facts: false
+  connection: local
+
+  tasks:
+    - name: run CVP provisioning
+      import_role:
+        name: arista.avd.eos_config_deploy_cvp
+      vars:
+        container_root: 'DC1'
+        configlets_prefix: 'AVD'
+        execute_tasks: false
+        apply_mode: strict
+        state: present
+        cv_collection: v3
+```
+
+You would execute it using the same command:
 
 ```shell
-ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml --tags provision
+ansible-playbook playbook.yml
 ```
 
 This will:
+
 - Create a CloudVision Configlet for each device in the inventory, containing the device's entire configuration.
 - Build out containers within CloudVision that represent the hierarchy of groups represented in the inventory.
 - Create tasks to apply the new configuration to each device within the inventory.
 
 At this point the changes can be reviewed, approved and executed in accordance with existing company requirements through CloudVision.
 
```

Comment:
Just a question more than expecting a change but with this new intro example there is a loss of info as compared to the previous ones regarding the tags build/provisision. is it expected? is this info somewhere else?

Reply:
You are right. This initial example is meant as the 2nd step for a complete Ansible AVD rookie, after reading the "Introduction to Ansible and AVD". More advanced options will come in later examples.